jobs/build/gen-assembly/Jenkinsfile

```diff
@@ -69,33 +69,14 @@ node {
 
         stage("gen-assembly") {
             withEnv(['KUBECONFIG=/home/jenkins/kubeconfigs/art-publish.app.ci.kubeconfig']) {
-                nightly_args = ""
-                for (nightly in commonlib.parseList(params.NIGHTLIES)) {
-                    nightly_args += " --nightly ${nightly.trim()}"
-                }
-                cmd = "--group openshift-${BUILD_VERSION} release:gen-assembly --name ${ASSEMBLY_NAME} from-releases ${nightly_args}"
-                if (params.CUSTOM) {
-                    cmd += ' --custom'
-                    if ((params.IN_FLIGHT_PREV && params.IN_FLIGHT_PREV != 'none') || params.PREVIOUS) {
-                        error("Specifying IN_FLIGHT_PREV or PREVIOUS for a custom release is not allowed.")
-                    }
-                }
-                else {
-                    if (params.IN_FLIGHT_PREV && params.IN_FLIGHT_PREV != 'none') {
-                        cmd += " --in-flight ${IN_FLIGHT_PREV}"
-                    }
-                    if (params.PREVIOUS) {
-                        for (previous in params.PREVIOUS.split(',')) {
-                            cmd += " --previous ${previous.trim()}"
+            withCredentials([string(credentialsId: 'ashwin-github-token', variable: 'GITHUB_TOKEN')]) {
```

Comment:
We'd need to use our art bot's github token here.. do take a look in prepare-release where we make commits to releases.yml, as to how we are authenticating to github there

Reply:
Is it [this](https://github.com/openshift/aos-cd-jobs/blob/master/jobs/build/prepare-release/Jenkinsfile) one? I could find only the JIRA token

Reply:
I found `openshift-bot-token` in `jobs/scanning/images-health/Jenkinsfile`. I'm hoping this is the one?

Reply:
Done!